pkg/discovery/endpoint/restapi/operations.go

```diff
@@ -186,6 +193,24 @@ func (o *Operation) webFingerHandler(rw http.ResponseWriter, r *http.Request) {
 		writeResponse(rw, resp, http.StatusOK)
 	case strings.HasPrefix(resource, fmt.Sprintf("%s%s", o.baseURL, o.webCASPath)):
 		o.handleWebCASQuery(rw, resource)
+	case resource == o.baseURL:
+		resp := &WebFingerResponse{
+			Subject:    "ipns://$publicKeyHashValue",
```

Comment:
What is this: $publicKeyHashValue?

Reply:
This placeholder to replace with ipfs public key hash

Reply:
I removed publicKeyHashValue placeholder